qubes-rpc/admin.repos.Enable

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+
+# Empty output indicates success; any input indicates error (probably an exception)
+
+import dnf
+import iniparse
+import sys
+
+base = dnf.Base()
+
+base.read_all_repos()
+
+reponame = sys.stdin.readline()
+repo = base.repos[reponame]
+
+# Loosely based on write_raw_configfile() from DNF source code,
+# because that method was introduced in DNF 2.0 but Qubes dom0 has DNF 1.x.
+with open(repo.repofile) as fp:
+	ini = iniparse.INIConfig(fp)
+
+ini[reponame]['enabled'] = 1
+
+with open(repo.repofile, 'w') as fp:
```

Comment:
This should write to a temporary file and rename it over original only when the whole file is written. Otherwise interrupting the operation (power failure, lack of disk space etc) would result in a broken (empty) repository configuration.